5_databases/week2/Part1.sql

```diff
@@ -0,0 +1,21 @@
+-- Active: 1699188605886@@0.0.0.0@3306@hyf_lesson1
+
+insert into task (id, title, description, created, updated, due_date, status_id, user_id) values (36, 'Catch every Pokemon', 'Catch them all', '1997-04-01 06:54:16', '2023-03-24 13:05:09', null, 2, 3);
+
+UPDATE task
+SET title = "Become a trillionaire"
+WHERE title = "Become a billionaire"
```

Comment:
Correct, but better to do with a primary key (id) so you dont edit more than one